### DISCUSSION TITLE
Dependencies in maven GAV notation

### DISCUSSION THREAD

**Discussion by gun2rin** [apache/camel-karavan#899] [Q&A] [answered]
Could you please explain what i am doing wrong when i am trying to set up postgres dependency at camel.jbang.dependencies?

I am using notation: 
groupId:artifactId:version

My camel.jbang.dependencies = camel-console, camel-jdbc, org.postgresql:postgresql:42.0.0

**And i am receiving the error:** 
Could not find artifact  org.postgresql:postgresql:jar:42.0.0 in central (https://repo1.maven.org/maven2)

What is the correct way to set up an external dependencies via application.properties ?

**Comment by mgubaidullin** [comment, marked as answer]
I think that it should not be spaces, like:
```
camel.jbang.dependencies=camel-console,camel-jdbc,org.postgresql:postgresql:42.0.0
```

@davsclaus fyi

**Comment by gun2rin** [comment]
@mgubaidullin thanx! it's Downloaded: org.postgresql:postgresql:42.6.0 (took: 964ms) from: central@https://repo1.maven.org/maven2  now ))

**Comment by davsclaus** [comment]
Yes you should not have spaces, something we likely can improve, but there are several places where artifacts are resolved, so its not just 1 code change.

**Comment by davsclaus** [comment]
I created a ticket
https://issues.apache.org/jira/browse/CAMEL-19890

The white-space is hard to spot in the log error message
